internal/cloudsql/refresh.go

```diff
@@ -110,7 +110,7 @@ func fetchEphemeralCert(ctx context.Context, client *sqladmin.Service, inst conn
 	}
 	resp, err := client.SslCerts.CreateEphemeral(inst.project, inst.name, &req).Context(ctx).Do()
 	if err != nil {
-		return tls.Certificate{}, fmt.Errorf("create ephemeral failed: %w", err)
+		return tls.Certificate{}, fmt.Errorf("create failed: %w", err)
```

Comment:
this seems way more vague - create what?

Reply:
This was a dev artifact. I've improved the error message.